console/src/main/scala/io/joern/console/Console.scala

```diff
@@ -12,8 +12,8 @@ import io.shiftleft.codepropertygraph.cpgloading.CpgLoader
 import io.shiftleft.semanticcpg.language.*
 import io.shiftleft.semanticcpg.language.dotextension.ImageViewer
 import io.shiftleft.semanticcpg.layers.{LayerCreator, LayerCreatorContext}
-import overflowdb.traversal.help.Doc
-import overflowdb.traversal.help.Table.AvailableWidthProvider
+import flatgraph.help.Doc
+import flatgraph.help.Table.AvailableWidthProvider
```

Comment:
[shared import name]

Can we add an indirection of some kind?

i.e. get some shared name that works for both overflowdb and flatgraph?